frontend/components/page-contents/feedback-task-content/RecordFeedbackTaskAndQuestionnaire.content.vue

```diff
@@ -409,6 +482,38 @@ export default {
         };
       }
     },
+    async searchRecords(
+      datasetId,
+      offset,
+      responseStatus,
+      searchText,
+      numberOfRecordsToFetch = 10
+    ) {
+      try {
+        const url = `/v1/me/datasets/${datasetId}/records/search?include=responses&response_status=${responseStatus}&limit=${numberOfRecordsToFetch}&offset=${offset}`;
+
+        const body = JSON.parse(
+          JSON.stringify({
+            query: {
+              text: {
+                q: searchText,
+              },
+            },
+          })
+        );
+
+        const { data } = await this.$axios.post(url, body);
```

Comment:
I think axios supports query params to be passed as an object. This can simplify the URL definition

```suggestion
        const url = `/v1/me/datasets/${datasetId}/records/search`;

        const body = JSON.parse(
          JSON.stringify({
            query: {
              text: {
                q: searchText,
              },
            },
          })
        );
?include=responses&response_status=${responseStatus}&limit=${numberOfRecordsToFetch}&offset=${offset}`;
        const { data } = await this.$axios.post(url, body, { params:  { include: "responses" , response_status: responseStatus, limit : numberOfRecordsToFetch, offset } );
```